### ISSUE TITLE
NoSuchMethodException on Main command

### ISSUE THREAD

**Issue by Maxetto** [games647/LagMonitor#22]
Doing /lagmonitor (or /lagmonitor help) in Spigot 1.8.8 reports a NoSuchMethodException about sendMessage( BaseComponent[] ):
http://pastebin.com/3bQ9JR8P

I belive, [THIS](https://github.com/games647/LagMonitor/blob/85eccb4961ff2d3f3a593bc47a6cd538ecc7ec4c/src/main/java/com/github/games647/lagmonitor/commands/HelpCommand.java#L79) line should be `sender.spigot().sendMessage(usageComponent);`

**Comment by games647** [comment]
Thanks. Yeah it looks like that they added the shortcut method only on later versions.

**Comment by Maxetto** [comment]
I reopen this because now it crashes the client with:
![parseexception](https://cloud.githubusercontent.com/assets/1360752/21639196/151e8f9a-d26f-11e6-8ef5-c0816e5ef5c6.png)

(Might be related with ProtocolLib v3.6.5 or one of my plugins in which I do (Hooking with ProtocolLib):
```
for (WrappedChatComponent component : chatComponents.getValues()) {
	if (component == null)
		return;
	String before = component.getJson();
```

**Comment by games647** [comment]
Could you try it without those?

**Comment by Maxetto** [comment]
Still happens even if no plugins

**Comment by Maxetto** [comment]
https://github.com/games647/LagMonitor/blob/e2b6a55f310d5362fb8a366fdea385a4dfa28dbc/src/main/java/com/github/games647/lagmonitor/commands/HelpCommand.java#L66

Shouldn't it be "\\n"?

**Comment by games647** [comment]
It doesn't make any difference

**Comment by games647** [comment]
Looks like a parsing bug in the 1.8 clients. It's fixed in 1.11, but I wrote a work-around now anyway.
